package/src/lib/octokit.ts

```diff
@@ -91,8 +99,11 @@ export const getGistFile = async (
     filename: string
     ) => {
   const gist = await gistGrabber(gistId);
-  const file = gist.files[filename];
-  return file ? file : null;
+  if (gist?.files){
```

Comment:
This thing is fidgety... lol. This is just because i finally was able to find the right Type for the Endpoint! There is a new version of it that exits early... instead of at the end.